arelle/ModelDocument.py

```diff
@@ -72,7 +72,7 @@ def load(modelXbrl, uri, base=None, referringElement=None, isEntry=False, isDisc
             modelXbrl.uriDir = os.path.dirname(modelXbrl.uriDir)
     if modelXbrl.modelManager.validateDisclosureSystem and \
        not normalizedUri.startswith(modelXbrl.uriDir) and \
-       not modelXbrl.modelManager.disclosureSystem.hrefValid(normalizedUri):
+       not modelXbrl.modelManager.disclosureSystem.hrefValidForValidTaxonomy(normalizedUri):
         blocked = modelXbrl.modelManager.disclosureSystem.blockDisallowedReferences
```

Comment:
I thought, for non-blocking, there was a desire to provide a warning or info notice if a href'ed taxonomy were not in the currently accepted list of taxonomies, in which case this logic might need some extra TLC?

Reply:
No we want all the other disclosure systems to remain the way they were. We want to add a singular disclosure system for `all-years` that can throw the EFM 6.22.00 errors.